.gitignore

```diff
@@ -4,3 +4,5 @@
 *.sw[nop]
 *~
 .DS_Store
+
+ejp-rd/README.md.backup
```

Comment:
Remove this.  If you think *.backup would be generally useful to ignore, please create another PR for that change.